.github/workflows/make-with-lints.sh

```diff
@@ -0,0 +1,3 @@
+make |& \
+	(! grep -v 'Warning: Found SVG with width or height specified, which will make the artwork not scale.  Specify a viewBox only to let the artwork scale.') | \
```

Comment:
I don't think we should have this `(! ... )` around this grep invocation, because it negates the exit status, and causes a failure. See this abbreviated example:

```
$ make clean && bash -c -o pipefail "make |& (! grep -v 'SVG with width') | (! grep -E 'Warning|Error')" && echo ok || echo fail
rm -f .tags .targets.mk issues.json \
    draft-irtf-cfrg-vdaf.{txt,html,pdf} index.html \
    draft-irtf-cfrg-vdaf-[0-9][0-9].{xml,md,org,txt,raw.txt,html,pdf} \
    draft-irtf-cfrg-vdaf.xml \
     diff-draft-irtf-cfrg-vdaf.html
fail
```